### PR TITLE
[PTFE] New admin user workflow

### PR DESCRIPTION
**Pull request by koikonom**

### CODE REVIEW

File: content/source/docs/enterprise/private/config.html.md [hunk 1]
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
...
+![PTFE Installer Dashboard](./assets/post-install-repl-console.png)
 
 ### Creating an Administrator
 
-After clicking on "Begin Installation" you will be brought to a page called
-"Create Administrator". Enter the requested information to create the first
-Administrator account. You will be able to create additional administrators
-later.
+After clicking on the "Open", right below the "Stop Now" button, you will
+be brought to a page asking you to create the first PTFE administrator account.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/config.html.md]: > After clicking "Open" below the button labeled "Stop Now", you'll see a page where you can create the first PTFE admin user.

Comment (armchairlinguist) [reply content/source/docs/enterprise/private/config.html.md]: As a side note - do we have any control over what the button says? "Open" is pretty vague.

File: content/source/docs/enterprise/private/config.html.md [hunk 2]
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
...
+You will be able to create additional administrators once you login.
+
+![New admin dialog](./assets/create-an-account.png)
 
 ### Creating an Organization
 
 The next step will create the first Organization.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/config.html.md]: Our general style is not to uppercase entity types - can we change this to "first organization" while we're in here?

File: content/source/docs/enterprise/private/config.html.md [hunk 3]
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
...
 
 ![Create a new organization](./assets/create-organization.png)
 
-If you skip this step, you will be prompted the create the first organization
-when you first log in.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/config.html.md]: Does this not apply anymore?

File: content/source/docs/enterprise/private/config.html.md [hunk 4]
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
...
-### Configure Twilio (optional)
-
-Terraform Enterprise uses Twilio to send SMS if a user selects SMS as their multi-factor
-authentication option. If Twilio is not configured they must use the Google
-Authenticator app on their smartphone instead.
-
-When you have added your configuration press "Save and Continue," or skip to the
-next step.
+After this is done, you can either continue with the creation of a new workspace,
+choose to configure other aspects of PTFE, or add more users.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/config.html.md]: I might change this to explicitly include a mention of "configure login settings such as SAML or Twilio", with links to the relevant configuration pages. (If we don't have existing details for Twilio, we'll need them. :)

File: content/source/docs/enterprise/private/config.html.md (hunk 3 above)
Comment (koikonom) [reply content/source/docs/enterprise/private/config.html.md]: There is no skip button. If you navigate elsewhere (user prefs, admin settings, whatever) and then click back on the TFE icon you'll be asked to create an org. If you log out and log in again, you'll be asked to create an org.

Comment (armchairlinguist) [reply content/source/docs/enterprise/private/config.html.md]: Oh, good call. I forgot that you're already logged in with this new flow.

**Comment by koikonom** [comment]
@armchairlinguist addressed most comments apart from the last one about twilio, because I had some questions but no time to ask :)

I am pushing this just in case it needs to go out while I am away. Please feel free to change as required, otherwise we can talk about it on Monday.

File: content/source/docs/enterprise/private/config.html.md [hunk 5]
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
...
-
-If you skip this step, you will be prompted the create the first organization
-when you first log in.
-
-After this is done, and showing the `Create a new Workspace` page, click back to the previous tab/browser window and continue.
-
-### Configure SMTP (optional)
+After clicking on the "Open", right below the "Stop Now" button, you will
+be brought to a page asking you to create the first PTFE administrator account.
+You will be able to create additional administrators once you login.
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/config.html.md]: once you login. > once you log in.

File: content/source/docs/enterprise/private/config.html.md [hunk 6]
```diff
@@ -23,64 +21,27 @@ correctly restored from backup. Please contact HashiCorp for help.
 In all examples below, be sure to replace "`<TFE HOSTNAME>`" with the hostname
 of your Private Terraform Enterprise instance.
 
-Navigate to `https://<TFE HOSTNAME>/admin/bootstrap` in your browser. You will
-be presented with a status page that looks like this:
+Navigate to `https://<TFE HOSTNAME>:8800/` in your browser. You will
+be presented with the installer's dashboard:
```
Comment (armchairlinguist) [inline content/source/docs/enterprise/private/config.html.md]: installer's > installer

**Comment by koikonom** [comment]
No worries that's why I keep asking you for feedback. I'd expect nothing less :P

**Pull request merged**